static/app/components/dropdownMenuItemV2.tsx

```diff
@@ -248,16 +264,26 @@ const MenuItem = withRouter(
 );
 export default MenuItem;
 
-const MenuItemWrap = styled('li')<{isDisabled?: boolean}>`
+const MenuItemWrap = styled('li')<{
+  isDisabled?: boolean;
+  isFocused?: boolean;
+  priority?: 'primary' | 'danger';
```

Comment:
should we create a` type Priority = 'primary' | 'danger'` and reuse it here and in the props?

Reply:
oh great point! done!